app/editor/src/components/formik/dropdown/FormikDropdown.tsx

```diff
@@ -1,11 +1,13 @@
+import { IOptionItem } from 'components';
 import { Dropdown, ISelectProps } from 'components/form';
 import { useFormikContext } from 'formik';
 
 import * as styled from './FormikDropdownStyled';
 
 export interface IFormikDropdownProps extends ISelectProps {
   name: string;
-  value?: string | number | readonly string[];
+  value?: string | number | readonly string[] | IOptionItem;
+  outerClassName?: string;
```

Comment:
I'm not a huge fan of this type of property `outerClassName`.  Indicates bad design implementation.   What functionality are you trying to achieve with this?

Reply:
It was to control some of the spacing after our label as `className` was controlling the actual input. Should I remove and just try to manipulate the `label` directly?

Reply:
The general rule of thumb is any classname or styling applies to the root/main element.   CSS can be used to selectively apply rules to children.